### PR TITLE
Geirst/fix disk and memory usage sampling for docker

### PR DESCRIPTION
**Pull request by geirst**
@toregge and @baldersheim please review

### CODE REVIEW

**Review by toregge — APPROVED** [review]
LGTM.

File: searchcore/src/vespa/searchcore/proton/server/disk_mem_usage_sampler.cpp [hunk 1]
```diff
@@ -59,10 +45,26 @@ DiskMemUsageSampler::sampleUsage()
...
+}
+
+}
+
 void
 DiskMemUsageSampler::sampleDiskUsage()
 {
-    _filter.setDiskStats(std::experimental::filesystem::space(_path));
+    _filter.setDiskStats(sampleDiskUsageInDirectory(_path));
 }
```
Comment (baldersheim) [inline searchcore/src/vespa/searchcore/proton/server/disk_mem_usage_sampler.cpp]: Check if disk is slow or fast

**Comment by baldersheim** [comment]
Fine, except from the comment.

**Comment by geirst** [comment]
@baldersheim please review again

**Review by baldersheim — APPROVED** [review]
👍

**Pull request merged**